argoverse/data_loading/object_label_record.py

```diff
@@ -11,10 +11,22 @@
 import numpy as np
 
 from argoverse.utils.calibration import CameraConfig
-from argoverse.utils.cv2_plotting_utils import draw_clipped_line_segment
+from argoverse.utils.cv2_plotting_utils import draw_clipped_line_segment, proj_cam_to_uv
 from argoverse.utils.se3 import SE3
 from argoverse.utils.transform import quat2rotmat
 
+BKGRND_RECT_OFFS_UP = 30 # px
+BKGRND_RECT_OFFS_DOWN = 10 # px
+
+BKGRND_RECT_OFFS_LEFT = 70 # px
+BKGRND_RECT_OFFS_RIGHT = 70 # px
+
+WHITE_BGR = (255,255,255)
+EMERALD_RGB = (80, 220, 100)
+BKGRND_RECT_ALPHA = 0.45
+
+TEXT_OFFS_LEFT = 70 # px
+MAX_RANGE_THRESH_PLOT_CATEGORY = 50 # meters
```

Comment:
Should this be consistent with the range we consider for the challenges?

Reply:
gets too messy beyond 50 m in my experience